docs/reference/modules/migration.md

```diff
@@ -8,12 +8,12 @@ extends: '@apostrophecms/module'
 
 <AposRefExtends :module="$frontmatter.extends" />
 
-This module provides services for database migrations. These **migrations are used to make changes to the database** at the time of a new code deployment, typically because of *data structure changes* in code or *to fix data errors*. This is completely separate from transferring data between environments or between versions of Apostrophe.
+This module provides services for database migrations. These **migrations are used to make changes to the database** at the time of a new code deployment, typically because of *data structure changes* in code or *to fix data errors*. This is completely separate from transferring data between environments or between major versions of Apostrophe.
 
-The `@apostrophecms/migration:migrate` task carries out all migrations that have been registered with this module, though typically only new migrations will run (see the warning below). In development environments the task function also runs on every site start up.
+The `@apostrophecms/migration:migrate` task carries out all migrations that have been registered with this module, though typically only new migrations will run (see the warning below). In development environments all new migrations also run on every every site startup.
 
 ::: warning
-**Migrations must be written so they are safe to run multiple times.** Apostrophe tracks when migrations have been run before in a the `aposMigrations` database collection, but there is no guarantee that they will not run again if that cache is cleared. If this is difficult to guarantee, you may wish to [write a task](/reference/module-api/module-overview.md#tasks-self) that executes the changes instead.
+**Migrations must be written so they are safe to run multiple times.** Apostrophe tracks when migrations have been run before in the `aposMigrations` database collection, but there is no guarantee that they will not run again if data is cleared the collection. If this is difficult to guarantee, you may wish to [write a task](/reference/module-api/module-overview.md#tasks-self) that executes the changes instead.
```

Comment:
"if data is cleared the collection" -> grammar

But I think I'm ready to be bold and just remove this paragraph! We track these in a real honest to god database collection now. Expecting code to work when that is gone for no reason is no more sensible than expecting it to work if half the records in aposDocs disappear for no reason.

Reply:
wow